spec/requests/atmosphere/api/virtual_machine_flavors_spec.rb

```diff
@@ -78,13 +78,13 @@
         end
       end
 
-      context 'when filter defined for compute site' do
+      context 'when filter defined for tenant' do
 
-        it "returns flavors at given cs only" do
-          get api("/virtual_machine_flavors?compute_site_id=#{cs.id}", user)
+        it "returns flavors at given tenant only" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.